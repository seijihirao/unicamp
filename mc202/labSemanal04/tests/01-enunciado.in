is-empty
pop
print-last
push   mas um dia
push desses   
push eu
print-first
print-last
shift
print-first
is-empty
exit

