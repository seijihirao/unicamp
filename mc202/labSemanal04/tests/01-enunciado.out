yep
mas um dia
eu
desses
nope
